projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -92,3 +83,26 @@ export async function jobRequest(endPoint: string) {
     }
   }
 }
+
+/**
+ * Sent a POST request to the server to get information about the data job of current directory
+ */
+export async function jobdDataRequest() {
```

Comment:
nit : Add a return type for the function, even if it is void.
https://www.typescriptlang.org/docs/handbook/2/functions.html#return-type-void